clang/test/Driver/ptrauth.c

```diff
@@ -0,0 +1,32 @@
+// Check that we can manually enable specific ptrauth features.
```

Comment:
aarch64-ptrauth.c is more conventional for such target-specific features

Reply:
Thanks, see f3abb4897fb9cd56b00eb6344489bbc972c40c5b and 4fd37cf125c13982ae0b6eb7254404fa3def6df5